docs/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: network.js
+title: Network JS
+version: 0.0
```

Comment:
`oz-docs init` should've set this up correctly I think... Not sure what happened.

```suggestion
version: 0.2
```

Reply:
https://github.com/OpenZeppelin/docs-utils/blob/dc7ce3006b6065cc4edebe53896644da5bd7dbec/oz-docs.js#L219-L229

Reply:
:man_shrugging: